Apr 30 2025 plugin releases (#843)
128technology/docs: docs/plugin_monitoring_agent.md

## Monitoring Agent Plugin Release Notes

### Release 4.1.0

**Release Date:** Apr 30, 2025

#### New Features and Improvements:

- **I95-58843** Version agnostic HTTP input for SSR input APIs

The monitoring-agent t128_http input prefers socket based HTTP APIs over legacy implementations.

- **PLUGIN-2628** The CPU collector performance has been improved for SSR >= 6.3

> The CPU collector will new perfer a static set of datapath and control cores to monitor to reduce gathering time for the data.

Reviewer: ```suggestion
The CPU collector now monitors a static set of datapath and control cores, reducing the collection time of the data.
```
